CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -2258,7 +2327,11 @@ end
 ----------------
 -- Scroll bar --
 ----------------
-function Methods.ApplyScrollbarToFrame(devConsole, scrollbar, window, body, frame)
+function Methods.ApplyScrollbarToFrame(devConsole, 
```

Comment:
So I don't know much about what's what with GUI widgets and git.
ScrollingFrame like 
http://wiki.roblox.com/index.php?title=API:Class/ScrollingFrame?

It seems to be a lua implementation of scroll logic, yes: renders the scroll bar, the thumb, etc.

I don't know why the person who implemented this chose to roll his own vs using what's there.